Modules/_sqlite/connection.h

```diff
@@ -39,6 +39,12 @@ typedef struct _callback_context
     pysqlite_state *state;
 } callback_context;
 
+enum autocommit_mode {
+    AUTOCOMMIT_COMPAT = LEGACY_TRANSACTION_CONTROL,
```

Comment:
```suggestion
    AUTOCOMMIT_LEGACY = LEGACY_TRANSACTION_CONTROL,
```

Wouldn't it be worth changing this (and its usages) too, for clarity and consistency, as well as the same reasons as the original?

Reply:
Definitely. Consistency FTW.